api/docs/release.dox

```diff
@@ -161,6 +161,8 @@ changes:
    signature between AArch64 and RISC-V.
  - Renamed dr_get_sve_vector_length() to dr_get_vector_length() to share function
    signature between AArch64 and RISC-V.
+ - Changed the -LL_miss_file option by adding a process ID field to the output. This
```

Comment:
s/-LL/drcachesim -LL/ to give context as this is not a core DR option.

Reply:
Thank you for pointing out the issues. I have completed the modifications as you suggested.